hack/deploy-test-appset.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash
-set -ex
+set -e
 
 root=$(dirname "${BASH_SOURCE[0]}")
 
 prevcontext=$(kubectl config current-context)
 kubectl config use-context kind-argocd-control-plane
 
-# TODO: Make this generate argo apps in all created clusters
+kubectl get applications -n argocd --no-headers | cut -d' ' -f1 | xargs kubectl -n argocd delete application
```

Comment:
Should we have `|| echo "Not found"` similarly to how we delete the application set below?

Reply:
We are getting application names, so unless something else deletes them in the meantime, I think this is fine as is.